e2e/env/common.go

```diff
@@ -53,6 +77,22 @@ func init() {
 		fmt.Printf("No value found in '%s': using default value %s\n", EnvVarTestedVersion, DefaultTestedVersion)
 	}
 
+	clusterIDPrefix = os.Getenv(EnvVarClusterIDPrefix)
```

Comment:
I was thinking if it would be better if we:
- added new `clusterid` variable
- brought the logic inside the `ClusterID()` function here instead
- made the `ClusterID()` function to just return the `clusterid` variable

I'm suggesting it because it feels weird to me to have different places to manage basically the same thing.

Reply:
I was thinking the same at some moment, but then I just went with the flow :) I'll do this.

Reply:
Done.